### ISSUE TITLE
Would there be a way to get a histogram of s3 access times from the information that Coiled collects?

### ISSUE THREAD

**Issue by rsignell** [coiled/feedback#290]
I often have workflows that are mostly thousands of reads to s3 for chunks of the same size.   I can see some of the variability on the Dask dashboard, but was wondering:

Is there might be a way to get the metrics for the tasks programmatically as a dataframe or something so I could look at the distribution, the tails of the distribution, etc.   ?

**Comment by hendrikmakait** [comment]
Hi, @rsignell! I'm curious about the intent behind your request. What problem(s) would you like to solve using task metrics? Since there's a plethora of _possible_ metrics, which ones would you be interested in?

**Comment by rsignell** [comment]
Yes, I'm trying to figure out how many chunks at a time I should request for each task, and it would be good to know the distribution of s3 access times within the task
